packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -2793,6 +2793,7 @@ class _RectangularSliderValueIndicatorPathPainter {
     assert(!sizeWithOverflow.isEmpty);
     const double edgePadding = 8.0;
     final double rectangleWidth = _upperRectangleWidth(labelPainter, scale, textScaleFactor);
+    center = parentBox.localToGlobal(center);
```

Comment:
Instead of resetting the center parameter, could you create a final variable here with a name that implies what changed, like globalCenter. It would also help to add a comment that explains why we're shifting to global coordinates here.

Reply:
Done